### PR TITLE
Vertical distance change

### PR DESCRIPTION
**Pull request by kellan-simiotics**
Differences in the vertical location should have a doubled effect on distance. Sessions will only the session starter should not expire.

### CODE REVIEW

File: src/Fullcount.sol [hunk 1]
```diff
@@ -474,7 +473,7 @@ contract Fullcount is EIP712 {
         SessionAtBat[nextSessionID] = atBatID;
     }
 
-    // TODO Change name of function as tokens are no longer staked?
+    // Possibly change name of function as tokens are no longer staked?
```
Comment (zomglings) [inline src/Fullcount.sol:476]: Can you retain the `TODO:` prefix. I regularly do `git grep TODO` and `git grep TODO(zomglings)`.

File: test/Fullcount.t.sol [hunk 2]
```diff
@@ -314,6 +314,45 @@ contract FullcountTest_startSession is FullcountTestBase {
 
         vm.stopPrank();
     }
+
+    function test_session_with_only_starter_does_not_expire() public {
```
Comment (zomglings) [inline test/Fullcount.t.sol:318]: Does this test fail on `main` branch?

**Review by zomglings — APPROVED** [review]
lgtm

Just some small stuff

File: test/Fullcount.t.sol (hunk 2 above)
Comment (kellan-simiotics) [reply test/Fullcount.t.sol:318]: Yes, it does

**Pull request merged**